test/test_sparse_csr.py

```diff
@@ -1282,6 +1284,40 @@ def test_direct_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(coo_sparse.to_sparse_csr().to_sparse_coo(), coo_sparse)
 
+    @dtypes(*floating_and_complex_types())
```

Comment:
Please add skipping of this test if `TEST_WITH_ROCM == True`.

Reply:
Done! Thanks